jenkins/github_helpers.py

```diff
@@ -300,7 +298,10 @@ def verify_upgrade_packages(self, pull_request):
             pull_request.set_labels('Ready to review')
```

Comment:
Change this with `Ready to Merge` as these PRs will be merged without manual review once auto-merge bot is configured.